src/equations/equations.jl

```diff
@@ -243,14 +243,26 @@ function cons2entropy end
     entropy2cons(w, equations)
 
 Convert the entropy variables `w` based on a standard [`entropy`](@ref) to the
-conserved variables for a given set of `equations`. 
-`u` is a vector type of the correct length `nvariables(equations)`. 
-Notice the function doesn't include any error checks for the purpose of efficiency, 
+conserved variables for a given set of `equations`.
+`u` is a vector type of the correct length `nvariables(equations)`.
+Notice the function doesn't include any error checks for the purpose of efficiency,
 so please make sure your input is correct.
 The inverse conversion is performed by [`cons2entropy`](@ref).
 """
 function entropy2cons end
 
+"""
+    flux(u, orientation_or_normal_direction, equations::AbstractEquations{1})
+
+Enables calling `flux` with a non-integer argument `orientation_or_normal_direction` for
+one-dimensional equations. Returns the value of `flux(u, 1, equations)` scaled by
+`orientation_or_normal_direction`.
+"""
+@inline function flux(u, orientation_or_normal_direction, equations::AbstractEquations{1})
```

Comment:
```suggestion
@inline function flux(u, normal_direction::AbstractVector, equations::AbstractEquations{1})
```
The docstring should be adapted as well.